src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/add_panel/add_panel_flyout.test.tsx

```diff
@@ -29,12 +29,20 @@ import { ContactCardEmbeddable } from '../../../../test_samples/embeddables/cont
 import { ContainerInput } from '../../../../containers';
 import { mountWithIntl as mount } from 'test_utils/enzyme_helpers';
 import { ReactWrapper } from 'enzyme';
-
+// eslint-disable-next-line
+import { coreMock } from '../../../../../../../../core/public/mocks';
 // @ts-ignore
 import { findTestSubject } from '@elastic/eui/lib/test';
 
-// eslint-disable-next-line
-import { coreMock } from '../../../../../../../../core/public/mocks';
+// @ts-ignore
```

Comment:
Can we please remove this `// @ts-ignore` and use TypeScript casting instead, if needed.